tools/doc/type-parser.js

```diff
@@ -18,7 +18,7 @@ const jsGlobalTypes = [
   'Array', 'ArrayBuffer', 'DataView', 'Date', 'Error', 'EvalError', 'Function',
   'Object', 'Promise', 'RangeError', 'ReferenceError', 'RegExp',
   'SharedArrayBuffer', 'SyntaxError', 'TypeError', 'TypedArray', 'URIError',
-  'Uint8Array',
+  'Uint8Array', 'Set'
```

Comment:
should this just go ahead and add Map, WeakSet, WeakMap, Symbol, etc?

Reply:
@ljharb We had all common globals here but unused ones were purged in 6ac3c44012cf0a52c3d586c4fcf473d98fce69ef

@boneskull Nit: These types are sorted alphabetically, so the `Set` should go after `'RegExp'`.

Reply:
that seems like it serves only to frustrate contributors in the future when they use normal parts of the language :-/ cc @trott